packages/features/bookings/Booker/Booker.tsx

```diff
@@ -139,12 +139,12 @@ const BookerComponent = ({
             <StickyOnDesktop
               key="meta"
               className={classNames(
-                "relative z-10 flex",
+                "relative z-10 flex [grid-area:meta]",
```

Comment:
This wrapper div was the cause of the week view not being scrollable, because the grid area was actually defined in the child. Setting that on the parent makes the grid function like it should.